modules/builds-use-custom-builder-image.adoc

```diff
@@ -17,6 +17,8 @@ your custom builder image to execute your custom build logic.
 
 .Procedure
 
+//Annotations - needs update?
+
```

Comment:
No.

It feels like there is an advanced module for custom strategy where we talk about exposing the Docker socket. That needs a real deprecation notice.